site/docs/guides/streaming.mdx

```diff
@@ -142,7 +82,7 @@ In this example, a function outputs words to the console immediately upon genera
 For more information, refer to the [chat sample](https://github.com/openvinotoolkit/openvino.genai/tree/master/samples/python/chat_sample/).
```

Comment:
1. The link is incorrect.
2. It should be moved to chat article

Reply:
Added links to Python and C++ chat samples to Chat Scenario page